src/Ball.js

```diff
@@ -48,22 +50,46 @@ export class Ball {
         RUN.results[STATES.infected]--
         RUN.results[STATES.recovered]++
         this.hasMovement = true
+        if (this.vx > 0) {
```

Comment:
please, add a comment explaining the following new block of code.

Reply:
Resolved with a private method with a meaningful name